package.json

```diff
@@ -64,7 +64,7 @@
     "src/**/*.scss": "stylelint"
   },
   "engines": {
-    "node": "8.11.3",
+    "node": "8.12.0",
```

Comment:
I think I just copied this over from the `spraygun-react` repo. Is this broken there too?

Reply:
Good call. I'll check